### ISSUE TITLE
Setting to prevent creation of new cookies

### ISSUE THREAD

**Issue by erikgeurts** [revive-adserver/revive-adserver#559]
I propose to introduce a application-wide setting that will prevent Revive Adserver from creating cookies on the user's computer, which could be useful in scenarios where there is a legal requirement.

We should be aware that several features will not work if cookies are not available, for example frequency capping, conversion tracking.

We should ensure that the cookies that store the session ID and installer data for the user interface still work.

**Comment by dabiathlet** [comment]
Hi! Is there anything new on this topic? I want to switch to Revive from Google Ad Manager but only if I can use it without cookies (on user's computer). I think there could be more publishers that could be interested in this scenario .

**Comment by stylefieber** [comment]
Absolutely. There is no need for cookies anymore. Just use session - or local storage.
